playbooks/roles/caasp4/templates/skuba-terraform.tfvars.j2

```diff
@@ -0,0 +1,60 @@
+# This file is managed by ansible and based on terraform.tfvars.ci.example from github.com/SUSE/skuba
+image_name = "{{ image_name }}"
+internal_net = "{{ internal_net }}"
+internal_subnet = "{{ internal_subnet }}"
+internal_router = "{{ internal_router }}"
+external_net = "{{ external_net }}"
+stack_name = "{{ stack_name }}"
+subnet_cidr = "{{ subnet_cidr }}"
+"dns_nameservers" = [
+    "8.8.8.8",
+    "8.8.8.4"
+]
+masters = 1
+workers = 2
+master_size = "m1.medium"
+worker_size = "m1.medium"
```

Comment:
Is that large enough? Can those two be a default role var?

Reply:
That's what the caasp team uses in the CI. added as default role var